egs/swbd/s5/local/rnnlm/run_rescoring.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
I'm not sure that I like the fact that the training script and rescoring script are separate.
It means that after you run the training it may not be clear how to run the rescoring.
IMO at the very least you should say, at the top of the training script, how to test it via
rescoring.